client/views/windows/help.tpl

```diff
@@ -49,16 +49,28 @@
 
 	<div class="help-item">
 		<div class="subject">
-			<kbd class="key-all">Alt</kbd><kbd class="key-apple">⌥</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd>
+			<span class="key-all">Alt+Down</span>
+			<span class="key-apple">⌥↓</span>
 		</div>
 		<div class="description">
-			<p>Switch to the previous/next window in the channel list.</p>
+			<p>Switch to the next window in the channel list.</p>
 		</div>
 	</div>
 
 	<div class="help-item">
 		<div class="subject">
-			<kbd class="key-all">Ctrl</kbd><kbd class="key-apple">⌘</kbd> + <kbd>K</kbd>
+			<span class="key-all">Alt+Up</span>
```

Comment:
I would keep arrows here too, it's pretty common.

Reply:
What does your OS commonly do?